RELEASE_NOTES.md

```diff
@@ -3,6 +3,19 @@ Release Notes
 
 v1.0.0-alpha.x
 ---------------
+### Breaking changes
+
+- Removed `Context.Access.kWriteMultiple` and
+ `Context.Access.kReadMultiple`access patterns, due to not having
+ coherent use cases.
+ [1016](https://github.com/OpenAssetIO/OpenAssetIO/issues/1016)
+
+### New Features
+
+- Added `Context.Access.kCreateRelated` access pattern, to support use
+  cases where host wants to specify a new related entity is created,
+  but differentiation based on traitset comparison is not possible.
```

Comment:
[fix]
The trait-set comparison rules were our implicit way of stating that the entity should be new before, which was un clear - so we added this access to be explicit. Instead of considering whether trait set matches when publishing, a Host should ask "is the reference I have for the specific thing i'm creating, or some other entity that I wish to create a new thing in relation to" and set this accordingly.  How about something like:

```suggestion
- Added `Context.Access.kCreateRelated` access pattern, to  indicate
  when a workflow specifically creates a new entity as a relation to an
  existing one.
```

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1036/commits/0df5a207aa7be7a37f52b73c31078fb1bc575a0c